cdap-ui/app/cdap/components/FileBrowser/index.js

```diff
@@ -190,14 +190,25 @@ export default class FileBrowser extends Component {
   }
   ingestFile(content) {
     let namespace = NamespaceStore.getState().selectedNamespace;
-
+    let {scope = false} = this.props;
     let params = {
       namespace,
       path: content.path,
       lines: 10000,
       sampler: 'first'
     };
 
+    if (scope) {
+      if (typeof scope === 'string') {
+        params.scope = scope;
+      } else if (typeof scope === 'boolean') {
```

Comment:
Can you help me understand more why `scope` here can be boolean, but also can be string? I saw `scope` being set to `true` in `Experiments/CreateView` component, but then in here we set `scope` to `mmds` if `scope` is true. Why not just set `scope` to `mmds` in the first place?

Reply:
So today wrangler accepts a scope as string (`mmds` or `modelId`) but tomorrow we want to have it as a flag which will accept a boolean value to give us a new workspace id for the same file vs give the same workspace id for the same file. Thats the reason this can be either or. When we add the flag in the wrangler backend it should be simple enough for UI to deprecate it.